language/vm/src/file_format.rs

```diff
@@ -1549,6 +1556,9 @@ pub fn dummy_procedure_module(code: Vec<Bytecode>) -> CompiledModule {
     let mut fun_def = FunctionDefinition::default();
     fun_def.code = code_unit;
 
+    // We start with an empty module which has an initially empty list
+    // of function signatures
+    assume!(module.function_signatures.is_empty());
```

Comment:
It seems to me that it would make sense for empty_module to provide a post_condition to this effect.

Reply:
https://github.com/facebookexperimental/MIRAI/issues/264